src/Controller.js

```diff
@@ -275,19 +322,8 @@ export default class Controller {
 	}
 
 	_listenCallback() {
-
 		this._listenCallbackID = requestAnimationFrame( this._listenCallback );
-
-		const value = this.getValue();
-
-		// Only update the DOM if the value has changed. Controllers that control non-primitive data
-		// types get updated every frame to avoid the complexity of comparing objects
-		if ( value !== this._listenValuePrev || Object( value ) === value ) {
-			this.updateDisplay();
-		}
-
-		this._listenValuePrev = value;
-
+		this.updateDisplay();
 	}
```

Comment:
I added these checks while testing a [three.js example](https://threejs.org/examples/#webgl_clipping_advanced) that had an uncheckable checkbox with `listen()`. I thought `listen()` was causing the checkbox to lose interactivity by updating the display every frame. The author's intent was actually to disable the controller by using a setter that could ignore assignment calls. 

This code appeared to "fix the bug" because the value of `input.checked` would be allowed to change, while in truth it was out of sync with `getValue()`, and was never forced to `updateDisplay()` because the actual value never changed between frames.